code/__HELPERS/_macros.dm

```diff
@@ -131,6 +133,10 @@
 
 #define isatom(A) (istype(A, /atom))
 
+#define issqueletic(A) (isskellington(A) || isskelevox(A)) //Yes, it is a word
```

Comment:
But clusterfack said I could use it.
And "skeletal" lost meaning to me after so many times.

If that one triggers people I'll need a word just as catchy.